src/raylib.cc

```diff
@@ -527,18 +531,22 @@ static PyObject* get_buffer(PyObject* self, PyObject* args) {
   ObjRef objref;
   int64_t size;
   SegmentId segmentid;
+  int64_t metadata_offset;
   if (!PyArg_ParseTuple(args, "O&O&", &PyObjectToWorker, &worker, &PyObjectToObjRef, &objref)) {
     return NULL;
   }
-  void* address = reinterpret_cast<void*>(const_cast<char*>(worker->get_buffer(objref, size, segmentid)));
+  void* address = reinterpret_cast<void*>(const_cast<char*>(worker->get_buffer(objref, size, segmentid, metadata_offset)));
   std::vector<npy_intp> dim({size});
```

Comment:
Should it be this?

	std::vector<npy_intp> dim({static_cast<npy_intp>(size)});

Not sure why I didn't use to get an error but now I do.